extensions/email/src/main/java/org/mule/extension/email/internal/commands/DeleteCommand.java

```diff
@@ -39,9 +39,11 @@
      * @param folderName the folder where the emails are going to be fetched
      * @param emailId    an optional email number to look up in the folder.
      */
-    public void delete(MuleMessage message, RetrieverConnection connection, String folderName, Integer emailId)
+    public void delete(List<Integer> emailIds,
```

Comment:
it doesn't make much sense to me to receive a list of emailIds. Most common case you just want to delete one. At most, there should be a delete-bulk operation.

Also notice that you didn't change the jdocs accordingly